### PR TITLE
Added Levitation Trap

### PR DESCRIPTION
**Pull request by ItsSimba1712**

### CODE REVIEW

File: data/murderrun/function/traps/levitation_trap/levitation_trap.mcfunction [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Particle
+	particle minecraft:falling_dust{block_state:{Name:white_concrete}} ~ ~ ~ 0.3 0.3 0.3 0 1 force @a[distance=..20]
```
Comment (Zandercross12) [inline data/murderrun/function/traps/levitation_trap/levitation_trap.mcfunction:2]: Can you change this to a purple block? Probably do like purple concrete or something

Comment (ItsSimba1712) [reply data/murderrun/function/traps/levitation_trap/levitation_trap.mcfunction:2]: No

Comment (Zandercross12) [reply data/murderrun/function/traps/levitation_trap/levitation_trap.mcfunction:2]: ok bro

**Review by Zandercross12 — APPROVED** [review]
I approve! 👍

**Pull request merged**